src/dev/src/app/datepicker/css-regression.html

```diff
@@ -5,11 +5,9 @@
   -->
 
 <div class="clr-example-1" clrForm>
-    <input type="date" clrDate>
+  <input type="date" clrDate>
 </div>
 
 <div class="clr-example-2" clrForm>
-    <div class="clr-example-input">
-        <input type="date" [(clrDate)]="date">
-    </div>
+  <input type="date" [(clrDate)]="date">
```

Comment:
Do you have a change for the gemini tests with this change?

Reply:
Yes, I threw it up to see what tests failed on Gemini instead of doing it locally, faster.